tools/build-keys-from-ts.js

```diff
@@ -21,7 +21,16 @@ const { promises: { writeFile } } = fs;
         }
     );
 
-    const mergedKeys = alphabetizeKeyInterfaces({ ...keys, ...jsxKeys });
+    const backwardCompatibleKeys = {
+        ExperimentalRestProperty: [
```

Comment:
Just wondering if there’s a way to avoid having this same code in three places?

Reply:
I’m just not a fan of needing to update three places if any one of them changes. It’s easy to lose track of that.

Reply:
I can just create a separate file and import it if you like. It will still be in two places however, as one is the build result, but that's better than three I imagine.

Reply:
Agreed. My main concern is just people forgetting to update. If one is autogenerated, then it’s not a concern for that one.

Reply:
Commit added...